cloud: add Performance and Plus storage (#20936)
pingcap/docs: tidb-cloud/size-your-cluster.md

The Standard storage type is applied automatically to new clusters hosted on AWS and created with TiDB versions v7.5.5, v8.1.2, v8.5.0, or later.

#### Performance and Plus storage

> The Performance and Plus storage provide higher performance and stability, with pricing that reflects their value. Currently, these two storage types are only available upon request for clusters deployed on AWS. To request this feature, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com) and click **Request Support**. Then, fill in "Apply for TiKV storage type" in the **Description** field and click **Submit**.

Reviewer: ```suggestion
The Performance and Plus storage provide higher performance and stability, with pricing that reflects these enhanced capabilities. Currently, these two storage types are only available upon request for clusters deployed on AWS. To request the Performance or Plus storage, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com) and click **Request Support**. Then, fill in "Apply for TiKV storage type" in the **Description** field and click **Submit**.
```

> The Performance and Plus storage offer greater performance and stability,with corresponding price differentiation to match value delivered. Currently, these 2 types are only available upon request for AWS clusters. To request this feature, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com) and click **Request Support**. Then, fill in "Apply for TiKV storage type" in the **Description** field and click **Submit**.

Reviewer 2: Add a space after the comma.

> The Performance and Plus storage offer greater performance and stability, with corresponding price differentiation to match value delivered. Currently, these 2 types are only available upon request for AWS clusters. To request this feature, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com) and click **Request Support**. Then, fill in "Apply for TiKV storage type" in the **Description** field and click **Submit**.

Reviewer 3: ```suggestion
The Performance and Plus storage provide higher performance and stability, with pricing that reflects their value. Currently, these two storage types are only available upon request for clusters deployed on AWS. To request this feature, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com) and click **Request Support**. Then, fill in "Apply for TiKV storage type" in the **Description** field and click **Submit**.
```

> **Note:**
>
> You cannot decrease the TiFlash node storage after the cluster creation.

> ### Tiflash node storage types

Reviewer 2: Upper case "F" for TiFlash

TiDB Cloud provides the following TiFlash storage types for [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) clusters hosted on AWS:

> - [Basic storage](#basic-storage)

Reviewer 3: ```suggestion
- [Basic storage](#basic-storage-1)
```

- [Plus storage](#plus-storage)

#### Basic storage

The Basic storage is ideal for most workloads, providing a balance between performance and cost efficiency. 

#### Plus storage

> Plus storage offer greater performance and stability,with corresponding price differentiation to match value delivered. Currently, this type is only available upon request for AWS clusters. To request this feature, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com) and click **Request Support**. Then, fill in "Apply for TiFlash storage type" in the **Description** field and click **Submit**.

Reviewer 2: Plus storage `offer` should be `offers`

> Plus storage offers greater performance and stability, with corresponding price differentiation to match value delivered. Currently, this type is only available upon request for AWS clusters. To request this feature, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com) and click **Request Support**. Then, fill in "Apply for TiFlash storage type" in the **Description** field and click **Submit**.

Reviewer 3: ```suggestion
The Plus storage provides higher performance and stability, with pricing that reflects the value it delivers. Currently, this storage type is only available upon request for clusters deployed on AWS. To request this feature, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com) and click **Request Support**. Then, fill in "Apply for TiFlash storage type" in the **Description** field and click **Submit**.
```

> The Plus storage provides higher performance and stability, with pricing that reflects the value it delivers. Currently, this storage type is only available upon request for clusters deployed on AWS. To request this feature, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com) and click **Request Support**. Then, fill in "Apply for TiFlash storage type" in the **Description** field and click **Submit**.

Reviewer: ```suggestion
The Plus storage provides higher performance and stability, with pricing that reflects these enhanced capabilities. Currently, this storage type is only available upon request for clusters deployed on AWS. To request it, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com) and click **Request Support**. Then, fill in "Apply for TiFlash storage type" in the **Description** field and click **Submit**.
```
